src/main/scala/org/bitcoins/core/crypto/ECPrivateKey.scala

```diff
@@ -85,9 +85,17 @@ object ECPrivateKey extends Factory[ECPrivateKey] {
     val decodedBase58 : Seq[Byte] = Base58.decode(base58)
     //Drop(1) will drop the network byte. The last 5 bytes are dropped included the checksum (4 bytes), and 0x01 byte that
     //is appended to compressed keys (which we implemented as the default option).
-    val trim = decodedBase58.drop(1).dropRight(5)
+    val trim = {
```

Comment:
Factor this { ... } into a function, whenever you use val trim = {...} it indicates you should be probably create a function for whatever is inside {...}